### PR TITLE
ES-827992-Convert Word to PDF Samples

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
<html>
<body>

Task Link | https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/826283
-- | --
Description | Saample added for Word Document to PDF Conversion

</body>
</html>

Regrads,
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/ASP-NET_images/NugetPackage.png
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/ASP-NET_images/NugetPackage.png:1]: Change snap with browse window

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in ASP.NET MVC
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in ASP.NET MVC**.
+
+## Steps to convert word document to PDF in C#:
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md:13]: word document should always be "Word document"

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Step 1: Create a new ASP.NET Web Application  project.
+
+![Create ASP.NET MVC application in Visual Studio](ASP-NET-MVC_images/CreateProjectforConversion.png)
+
+Step 2: Select the MVC application
+
+![Create ASP.NET MVC application in Visual Studio](ASP-NET-MVC_images/MVC.png)
+
+Step 3: Install the [Syncfusion.DocIO.AspNet.Mvc5](https://www.nuget.org/packages/Syncfusion.DocIO.AspNet.Mvc5) and [Syncfusion.DocToPdfConverter.AspNet.Mvc5](https://www.nuget.org/packages/Syncfusion.DocToPdfConverter.AspNet.Mvc5) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md:23]: Installing DocToPDFConverter NuGet is enough, it install the DocIO as dependent

File: File-Formats/DocIO/ASP-NET-MVC_images/NugetPackage.png
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/ASP-NET-MVC_images/NugetPackage.png:1]: Change screenshot to install DocToPDFConverter

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Step 4: Include the following namespace in that HomeController.cs file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+using Syncfusion.Pdf;
+using System.Web.Mvc;
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md:38]: Remove the namespaces which not related to product

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+</div>
+}
+Html.EndForm();
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 7: Add a new action method **ConvertWordDocumentToPdf** in HomeController.cs and include the below code snippet to **convert the Word document to Pdf** and download it.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md:66]: Pdf should be "PDF"

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Open the file as Stream
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md:72]: Modify the code snippet in optimized way

File: File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+Step 1: Create a new ASP.NET Web application project.
+
+![Create ASP.NET Web application in Visual Studio](ASP-NET_images/CreateProjectforConversion.png)
+
+Step 2: Select the Empty project
+
+![Create ASP.NET Web application in Visual Studio](ASP-NET_images/ASPNET.png)
+
+
+Step 3: Install the [Syncfusion.DocIO.AspNet](https://www.nuget.org/packages/Syncfusion.DocIO.AspNet) and [Syncfusion.DocToPDFConverter.AspNet](https://www.nuget.org/packages/Syncfusion.DocToPDFConverter.AspNet) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md:24]: Address the same feedback like above

File: File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Create Word document in ASP.NET
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in ASP.NET Web Forms**.
+
+## Steps to convert word document to PDF in C#:
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md:13]: Word document

File: File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+{% endtabs %}
+
+Step 7: Include the below code snippet in the click event of the button in **MainPage.aspx.cs**, to **convert the Word document to Pdf** and download it.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Open an existing Word document.        
+string filePath = Server.MapPath("~/App_Data/Template.docx");
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md:82]: optimize the code

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/ASP-NET_images/NugetPackage.png
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/ASP-NET_images/NugetPackage.png:1]: Updated sir

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md]: Updated sir.

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md]: Updated sir

File: File-Formats/DocIO/ASP-NET-MVC_images/NugetPackage.png
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/ASP-NET-MVC_images/NugetPackage.png:1]: ScreenShot Changed

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md (hunk 3 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md]: Removed unwanted Namespaces sir

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md (hunk 4 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md]: Changed sir

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md (hunk 5 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md:71]: code snippet in optimized

File: File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md (hunk 7 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md]: Updated sir

File: File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md (hunk 8 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Documnet-to-PDF-ASP-NET.md:82]: Code optimized sir.

**Pull request merged**